src/org/myworldgis/io/geojson/GeoJsonReader.java

```diff
@@ -23,10 +23,8 @@
 import org.json.simple.parser.JSONParser;
 import org.json.simple.parser.ParseException;
 
-public class GeoJsonReader {
+public class GeoJsonReader implements GeoJsonConstants {
```

Comment:
That's an interest way to bring these constants in, I don't think I've seen that before.  It works for me.

Reply:
I found it done that way in the shapefile importer code and thought I would keep it consistent.